lib/modules/datasource/dotnet-version/index.ts

```diff
@@ -23,6 +23,14 @@ export class DotnetVersionDatasource extends Datasource {
     'https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/releases-index.json',
   ];
 
+  // De-comment when supporting PR which adds this feature is merged into main
+  // override releaseTimestampSupport = true;
+  // override releaseTimeStampNote =
+  //   'The release timestamp is determined from the `release-date` field in the results.';
+  override readonly sourceUrlSupport = 'package';
+  override readonly sourceUrlNote =
+    'We use the url <https://github.com/dotnet/sdk> for `dotnet-sdk` and, <https://github.com/dotnet/runtime> for the `dotnet-runtime`.';
```

Comment:
Let's not enclose urls inside angle brackets, let it be close to Markdown

```suggestion
    'We use the url https://github.com/dotnet/sdk for `dotnet-sdk` and, https://github.com/dotnet/runtime for the `dotnet-runtime`.';
```